includes/class-wc-calypso-bridge-woocommerce-admin.php

```diff
@@ -35,6 +35,39 @@ public static function factory() {
 	 */
 	public function init() {
 		add_filter( 'wc_admin_get_feature_config', array( $this, 'maybe_remove_devdocs_menu_item' ) );
+		add_action( 'admin_init', array( $this, 'redirect_store_details_onboarding' ) );
+	}
+
+	/**
+	 * Handle the store location's onboarding redirect when user provided a full address.
+	 *
+	 * @since 1.9.4
+	 * @return void
+	 */
+	public function redirect_store_details_onboarding() {
```

Comment:
`class-wc-calypso-bridge-setup.php` seems like a better place for this -- you may also be able to clean up a bunch of stuff there now that the Profiler is gone.